docs/params.ipynb

```diff
@@ -0,0 +1,638 @@
+{
```

Comment:
<p>I think we need one final example of running a sweep with this type of sweep to ensure that it works as expected. Presumably: </p><p><code>sim.run_sweep(c_flat, repetitions=20, params=new_sweep)</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5437/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>Done</p>